core-js/src/main/javascript/cdf-legacy/Dashboards.Utils.js

```diff
@@ -26,6 +26,12 @@ Dashboards.escapeHtml = function(input) {
   .replace(/"/g,"&#34;");
 };
 
+Dashboards.sanitizeHtml = function(html) {
+  // here is iframe explicitly replaced by script to further sanitizing since sanitizer itself doesn't sanitize iframe tag
+  html = html.replace(/<iframe\b[^>]*>/gi, "<script>").replace(/<\/iframe>/gi, "</script>");
```

Comment:
Still would allow <IfRaMe>, <[%00]iframe , <i[%00]fRame . Even this might make it's way through -> %3eiframe . Not sure that this is as effective as you might hope.

Reply:
`<IfRaMe>` will not pass as the regular expression is run with the `gi` flags — `i` stands for insensitive. As for the other combinations that you mention, how are they dangerous? Does a browser handle `<i[%00]fRame` as if it were an `iframe`?